modo/cram.py

```diff
@@ -5,86 +5,116 @@
 from pysam import (
     AlignedSegment,
     AlignmentFile,
+    VariantRecord,
 )
 from urllib.parse import urlparse
 import modo_schema.datamodel as model
 
 import sys
+import re
 import htsget
-from .helpers import parse_region, bytesio_to_alignment_segments
-from io import BytesIO
+from .helpers import (
+    parse_region,
+    get_fileformat,
+    make_pysam_inFile,
+    bytesio_to_iterator,
+    iter_to_file,
+)
+from io import BytesIO, StringIO
```

Comment:
```suggestion
from io import BytesIO
```

remove unused import of StringIO